### PR TITLE
feat: Add option to set default branch

### PR DESCRIPTION
**Pull request by abeatrix**
A feature frequently requested by our users

### CODE REVIEW

**Comment by casassg** [comment]
This would effectively solve the issue we are seeing

File: README.md [hunk 1]
```diff
@@ -27,6 +27,7 @@ This extension contributes the following settings:
 
 - `sourcegraph.url`: The Sourcegraph instance to use. Specify your on-premises Sourcegraph instance here, if applicable.
 - `sourcegraph.remoteUrlReplacements`: Object, where each `key` is replaced by `value` in the remote url.
+- `sourcegraph.setDefaultBranch`: String to set the name of the default branch. Always open files in the default branch.
```
Comment (muratsu) [inline README.md]: I think `sourcegraph.defaultBranch` is a better property name for this

Comment (abeatrix) [reply README.md]: Updated option to defaultBranch from setDefault Branch, thank you!

**Review by tjkandala — COMMENTED** [review]
Awesome! Can you add `sourcegraph.defaultBranch` to our [configuration schema](https://sourcegraph.com/github.com/sourcegraph/sourcegraph-vscode@eddb76166fe41fe3f788bb62adba8873a881904b/-/blob/package.json#L49-69) in the manifest?

**Comment by abeatrix** [comment]
@tjkandala Updated! Thanks for catching that!

**Review by tjkandala — APPROVED** [review]

**Pull request merged**